### ISSUE TITLE
算法、图像增强问题

### ISSUE THREAD

**Issue by windowxiaoming** [PaddlePaddle/PaddleOCR#15978]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

我尝试对华为文字进行标注，训练文本检测的算法时候，参数如下，分别在训练集和验证集加入同一张图片，发现识别的时候，漏检问题，各版本都遇到了。

问题描述：标注的时候如果不同的类别有IOU交叉部分，那么训练的模型，在测试模型识别效果会有丢失，而且丢失始终是后面的锚框

### 🏃‍♂️ Environment (运行环境)

Global:
  model_name: PP-OCRv5_server_det
  debug: false
  use_gpu: true
  epoch_num: 200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output_00/PP-OCRv5_server_det
  save_epoch_step: 5
  eval_batch_step:
  - 0
  - 6000
  cal_metric_during_train: false
  checkpoints: null
  pretrained_model: null
  save_inference_dir: /data1/XXX/projects/ocr/PaddleOCR-30/output_00/PP-OCRv5_server_det
  use_visualdl: false
  infer_img: /data1/XXX/projects/ocr/ppocr_train/training_data/det/train/
  save_res_path: ./output_00/det_db/predicts_db.txt
  distributed: true
Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: PPHGNetV2_B4
    det: true
  Neck:
    name: LKPAN
    out_channels: 256
    intracl: true
  Head:
    name: PFHeadLocal
    k: 50
    mode: large
Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0007
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 1e-6
PostProcess:
  name: DBPostProcess
  thresh: 0.5
  box_thresh: 0.8
  max_candidates: 1000
  unclip_ratio: 1.5
Metric:
  name: DetMetric
  main_indicator: hmean
Train:
  dataset:
    name: SimpleDataSet
    data_dir: /data1/XXX/projects/ocr/ppocr_train/training_data/det/
    label_file_list:
    - /data1/XXX/projects/ocr/ppocr_train/training_data/det/train.txt
    ratio_list:
    - 1.0
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - CopyPaste: null
    - IaaAugment:
        augmenter_args:
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 2
    - EastRandomCropData:
        size:
        - 960
        - 960
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: 200
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: 200
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: false
    batch_size_per_card: 2
    num_workers: 1
    use_shared_memory: true
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: /data1/XXX/projects/ocr/ppocr_train/training_data/det/
    label_file_list:
    - /data1/XXX/projects/ocr/ppocr_train/training_data/det/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest:
        limit_side_len: 1920
        limit_type: resize_long
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 1
    num_workers: 1
    use_shared_memory: true
profiler_options: null

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

<img width="530" height="117" alt="Image" src="https://github.com/user-attachments/assets/4e48e1df-acef-4317-a6a3-3072474161ac" />
<img width="547" height="151" alt="Image" src="https://github.com/user-attachments/assets/c38c59ae-0270-45b7-a06c-6ac58b2887e8" />

期待大佬们的包括但不限于技术回复和指导

**Comment by cuicheng01** [comment]
您好，这块是否可以提供更多细致的描述呢？

**Comment by leo-q8** [comment]
@windowxiaoming 您好，训练的数据量是多少呢，能否提供一下训练数据我们复现一下

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
